src/matrix/matrixfdist.jl

```diff
@@ -0,0 +1,142 @@
+"""
+```julia
+MatrixFDist(n1, n2, B)
+
+n1::Real  degrees of freedom (greater than p - 1)
+n2::Real  degrees of freedom (greater than p - 1)
+B::PDMat  p x p scale
+```
+The [matrix *F*-Distribution](https://projecteuclid.org/euclid.ba/1515747744)
+(sometimes called the matrix beta type II distribution) generalizes the
+*F*-Distribution to ``p\\times p`` real, positive definite matrices ``\\boldsymbol{\\Sigma}``.
+If ``\\boldsymbol{\\Sigma}\\sim MF_{p}(n_1/2,n_2/2,\\mathbf{B})``, then its probability density function is
+
+```math
+f(\\boldsymbol{\\Sigma} ; n_1,n_2,\\mathbf{B}) =
+\\frac{\\Gamma_p(\\frac{n_1+n_2}{2})}{\\Gamma_p(\\frac{n_1}{2})\\Gamma_p(\\frac{n_2}{2})}
+|\\mathbf{B}|^{n_2/2}|\\boldsymbol{\\Sigma}|^{(n_1-p-1)/2}|\\mathbf{B}+\\boldsymbol{\\Sigma}|^{-(n_1+n_2)/2}.
+```
+
+If the joint distribution ``p(\\boldsymbol{\\Psi},\\boldsymbol{\\Sigma})=p(\\boldsymbol{\\Psi})p(\\boldsymbol{\\Sigma}|\\boldsymbol{\\Psi})``
+is given by
+
+```math
+\\begin{align*}
+\\boldsymbol{\\Psi}&\\sim W_p(n_1, \\mathbf{B})\\\\
+\\boldsymbol{\\Sigma}|\\boldsymbol{\\Psi}&\\sim IW_p(n_2, \\boldsymbol{\\Psi}),
+\\end{align*}
+```
+
+then the marginal distribution of ``\\boldsymbol{\\Sigma}`` is
+``MF_{p}(n_1/2,n_2/2,\\mathbf{B})``.
+"""
+struct MatrixFDist{T <: Real, TW <: Wishart} <: ContinuousMatrixDistribution
+    W::TW
+    n2::T
+    logc0::T
+end
+
+#  -----------------------------------------------------------------------------
+#  Constructors
+#  -----------------------------------------------------------------------------
+
+function MatrixFDist(n1::Real, n2::Real, B::AbstractPDMat)
+
```

Comment:
I won't comment on this everywhere, but generally speaking we don't include blank lines between the function signature and the first statement, as well as between the last statement and the closing `end`. If you could fix those, that would be great.

Reply:
Done. Thanks!